### ISSUE TITLE
Release Checklist 22.10

### ISSUE THREAD

**Issue by caryr35** [NVIDIA/spark-rapids#6255]
Release Checklist for v22.10:

| Status  | Task | Start | End | Owner| 
| ------------- | ------------- | ------------- | ------------- | ------------- | 
|**Pre-release**| | | | 
|<ul><li>- [X] </li></ul> | Notification to team of last feature sprint in milestone  | 2022-09-05 | 2022-09-26  | @sameerz| 
|<ul><li>- [X] </li></ul> | Burndown (no new features for current release) | 2022-09-26 | 2022-10-07 | Team|  
|<ul><li>- [X] </li></ul> | Update docs for current release in spark-rapids without updating release links |2022-10-03 | 2022-10-07 | Team| 
|<ul><li>- [X] </li></ul> | Create a PR to main to update release links |2022-10-03 | 2022-10-07 | @nvliyuan | 
|<ul><li>- [X] </li></ul> | Create a PR to branch-22.12 to update release links |2022-10-03 | 2022-10-07 | @nvliyuan | 
|<ul><li>- [X] </li></ul> | Get RAPIDS 22.12 cudf java build working.  |2022-10-03 | 2022-10-09 | @pxLi | 
|<ul><li>- [X] </li></ul> | Create 22.12 branch including spark-rapids-jni and spark-rapids | 2022-9-28 | 2022-9-30  | @pxLi | 
|<ul><li>- [X] </li></ul> | Automerge from 22.10 to 22.12 | 2022-9-28 | 2022-9-30 | @pxLi | 
|<ul><li>- [X] </li></ul> | Enable Jenkins builds (premerge, nightly, integration, performance) for 22.12 | 2022-9-28 | 2022-9-30 | @pxLi | 
|<ul><li>- [X] </li></ul> | Virtual review | 2022-10-10 | 2022-10-21 |@krajendrannv | 
|<ul><li>- [X] </li></ul> | [spark-rapids-jni 22.10] Java release to sonatype | 2022-10-14 | 2022-10-14 | @NvTimLiu | 
|<ul><li>- [X] </li></ul> | [cudf 22.10] Java release to sonatype | 2022-10-14 | 2022-10-14 | @NvTimLiu | 
|<ul><li>- [X] </li></ul> | Notify RAPIDS team of javadocs for 22.10 | 2022-10-17 | 2022-10-17 | @GaryShen2008 | 
|**Staging release**||||
|<ul><li>- [X] </li></ul> | Make 22.12 the default branch | 2022-10-10 | 2022-10-10 | @jlowe | 
|<ul><li>- [X] </li></ul> | Merge all remaining branch-22.10 PRs | 2022-10-07 | 2022-10-07 | Team | 
|<ul><li>- [X] </li></ul> | Update spark-rapids-jni dependency version |2022-10-14  | 2022-10-14 | @pxLi | 
|<ul><li>- [X] </li></ul> | Create CHANGELOG.md | 2022-10-08 | 2022-10-08 | @pxLi | 
|<ul><li>- [X] </li></ul> | Deliver Snapshot packages to QA | 2022-10-10 | 2022-10-10 | @NvTimLiu| 
|<ul><li>- [X] </li></ul> | Create a PR (not squashed) to merge from branch-22.10 to main. Needs review from repository owner | 2022-10-14  | 2022-10-14 | @NvTimLiu| 
|<ul><li>- [X] </li></ul> | Trigger the release Jenkins job on the main branch and deploy the jar to sonatype staging | 2022-10-18 | 2022-10-18 | @NvTimLiu| 
|<ul><li>- [X] </li></ul> | Deliver final packages to QA | 2022-10-18  | 2022-10-18 | @NvTimLiu| 
|<ul><li>- [X] </li></ul> | Run integration tests on staging jars with AQE on, UCX on, Databricks Azure, Databricks AWS, Dataproc, CDH | 2022-10-18 | 2022-10-21 | @NvTimLiu| 
|<ul><li>- [X] </li></ul> | Run LHA tests on staging jars | 2022-10-18 | 2022-10-21 | @NvTimLiu| 
|<ul><li>- [X] </li></ul> | QA testing with Snapshot on YARN, NGC with Volta and Ampere | 2022-10-10 | 2022-10-18 | @yuange98|  
|<ul><li>- [X] </li></ul> | QA testing of Qualification tool UI | 2022-10-10 | 2022-10-18 | @yuange98|  
|<ul><li>- [X] </li></ul> | Run pre-processing workflow for [DLRM sample use case](https://github.com/NVIDIA/DeepLearningExamples/tree/master/PyTorch/Recommendation/DLRM#advanced)|2022-10-18 |2002-10-21 | @johnzhong |  
|<ul><li>- [X] </li></ul> | QA testing with final packages on YARN, NGC with Volta and Ampere | 2022-10-18 | 2022-10-21 | @yuange98|  
|**Release**| | | | |  
|<ul><li>- [X] </li></ul> | Tag the release and deploy the staging artifacts to maven central repository | 2022-10-26 | 2022-10-26 | @NvTimLiu|
|<ul><li>- [X] </li></ul> | Update gh-pages | 2022-10-26 | 2022-10-26 | @nvliyuan    |  
|<ul><li>- [X] </li></ul> | Test [spark-rapids-examples repo](https://github.com/NVIDIA/spark-rapids-examples) with Spark-Rapids 22.10 | 2022-10-24| 2022-10-26  | @nvliyuan  | 
|<ul><li>- [X] </li></ul> | Update docs for current release in spark-rapids-examples | 2022-10-24  | 2022-10-26  | @nvliyuan |
|<ul><li>- [X] </li></ul> | Update rapids.sh in https://github.com/GoogleCloudDataproc/initialization-actions/tree/master/rapids for the plugin (https://github.com/GoogleCloudDataproc/initialization-actions/pull/1026)| 2022-10-24 | 2022-10-26  | @nvliyuan  |

**Comment by sameerz** [comment]
Hi @sinkinben , please leave this issue pinned in the repository, thanks!

**Comment by sinkinben** [comment]
> Hi @sinkinben , please leave this issue pinned in the repository, thanks!

Sorry, sometimes my mac may get stuck, when switching the screen, I may accidentally touch/click some buttons. I will take care of this.

**Comment by pxLi** [comment]
Per CN holiday schedule (Oct 1. to Oct 7.),

I will create 22.12 branch and enable automerge and premerge CI to unblock new 22.12 feature development next week,

Some tasks relied on cudf could be postponed from Oct. 7 to Oct. 8-9 (weekends). So it will not actually delay the plan for following weekdays.
